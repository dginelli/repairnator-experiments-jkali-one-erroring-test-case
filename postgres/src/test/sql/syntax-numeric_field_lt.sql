SELECT assert(count(*), 35256, 'syntax-numeric_field_lt') FROM so_posts WHERE zdb('so_posts', ctid) ==> 'answer_count<20';
