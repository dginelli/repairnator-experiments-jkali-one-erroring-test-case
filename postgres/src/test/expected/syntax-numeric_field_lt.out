SELECT assert(count(*), 35256, 'syntax-numeric_field_lt') FROM so_posts WHERE zdb('so_posts', ctid) ==> 'answer_count<20';
 assert 
--------
 t
(1 row)

